src/commands/install.ts

```diff
@@ -459,7 +459,7 @@ export async function install(
   return {success: true, importMap};
 }
 
-export async function getInstallTargets(config: SnowpackConfig, scannedFiles?: [string, string][]) {
+export async function getInstallTargets(config: SnowpackConfig, scannedFiles?: SnowpackFile[]) {
```

Comment:
This is a good example of what this buys us: a more predictable structure. Here, `[string, string][]` is replaced with `SnowpackFile[]`.

Reply:
💥